setup.py

```diff
@@ -48,6 +48,8 @@ def exec_file(path_segments, name):
         "canonicaljson>=1.0.0",
         "unpaddedbase64>=1.0.1",
         "pynacl>=0.3.0",
+        "typing_extensions>=3.5.0",
+        "typing>=3.5",
```

Comment:
I believe you only want this if you're running under a Python that is older than 3.5.

At least it lead to some breakage in a couple of my projects.

For example: https://gitlab.matrix.org/matrix-org/olm/commit/0883a922ff39981e27cff7e677a6f79a5f324fab

Reply:
yup, good spot!